### ISSUE TITLE
Benchmarking single-cell RNA-sequencing protocols for cell atlas projects

### ISSUE THREAD

**Issue by ipediez** [ebi-ait/hca-ebi-wrangler-central#718]
### Project short name:
BenchmarkingSingleCellProtocols

### Primary Wrangler:
Ami

### Secondary Wrangler:
Ida
### Associated files

* Google Drive:
https://drive.google.com/drive/folders/10XD1BCC9ohFHVcNFGGTmUtJoIhoBuvw3

* Google Sheet:
https://docs.google.com/spreadsheets/d/1mhL-LOvfG67GfEdf1fBf0ch-fbqQ5s5hwqjyb1DEyaY/edit#gid=1503327377

* Ingest: https://contribute.data.humancellatlas.org/projects/detail?uuid=6e177195-0ac0-468b-99a2-87de96dc9db4

### Published study links

* Paper: https://doi.org/10.1038/s41587-020-0469-4

* Accessioned data: [GSE133549](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE133549)

### Key Events

- [x] Convert published metadata to HCA spreadsheet
- [x] Manually curate dataset to meet HCA metadata standard
- [x] Collect any matrix and cell-type annotation files
- [x] Upload sheet to validate metadata
- [ ] Check linking using ingest graph validator
- [x] Transfer raw files to ingest to validate data files
- [x] Ask the Secondary Wrangler for an end-to-end review of the project. Ask the Expertise Wrangler to review specific tabs if needed
- [ ] Submit dataset to Production 
- [ ] Complete the Export SOP
- [ ] Convert project data to SCEA format following the SCEA conversion SOP if appropriate

**Comment by ami-day** [comment]
Requires a new library preparation method ontology term. Request is here: https://github.com/HumanCellAtlas/ontology/issues/107

Requires library preparation method info. from the authors (various methods). I have emailed the authors to ask for this.

**Comment by ofanobilbao** [comment]
Moving this back to Wrangling while Ami is away, as I am not sure if this is ready for secondary review or not, as I have not seen any posts asking for review and not all the boxes in the acceptance criteria are ticketed.

**Comment by idazucchi** [comment]
@gabsie and @ami-day to dicuss if this will be included in the DCP

**Comment by Wkt8** [comment]
@gabsie what is the status of this?

**Comment by ofanobilbao** [comment]
@gabsie to investigate

**Comment by gabsie** [comment]
OK, on this project - let's just make a decision, this has been sitting here forever. 
The pipelines are no longer an issue. Let's publish the project with both raw and matrices. 
Any last issues from the rest of the wranglers? @ESapenaVentura @idazucchi @Wkt8

**Comment by gabsie** [comment]
OK - let's do it - both raw and matrices go into and we publish it. 
@ami-day  - please proceed

**Comment by idazucchi** [comment]
If the spreadsheet is ready for secondary review we can get it ready for R20, if not it will be postponed to R21
@ami-day

**Comment by ami-day** [comment]
`064f96e3-2af9-43b0-967e-b86cdd876e79`

**Comment by gabsie** [comment]
The majority of files has been downloaded.
Ida will finish secondary review.
This will probably not make it to release 21, but good to finalise it. @ami-day

**Comment by ami-day** [comment]
Synced data files. Graph validating

**Comment by ami-day** [comment]
Waiting on validation of metadata schema test data (cell suspension -> cell suspension as experiment design)

**Comment by idazucchi** [comment]
Hi ami, I'm done with the review. 
This is a large dataset with a lot of libraries and cell suspensions. I think we can simplify the way we model the experimental design and still be accurate by :
- using the same cell suspension `reference_sample` as input for all the libraries and files to highlight that they are all using the same cell mix. 
- delting the libarary specific enrichment and adding a few lines to the library protocol description if needed. There is no enrichment that requires them


### Project
- [GSE141469](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE141469) is in the project but all the files and library are missing from the spreadsheet
- some organs are missing
- the organisms are missing
- the ontology label and id for data curator are in the wrong fields
- the publication url points to the abstract

### Donor
- the human PBMCs donor are 2 female and 2 male
- `human adult stage` can't be used for non human donors
- `MDCK_donor` should be female, species  canis lupus familiaris

### Collection
- a protocol is needed to connect the cell line donors to the specimens

### Specimens
- the cell line specimens have no collection protocol

### Cell line
- you can add the publication information
- there is no protocol to connect the cell lines to their specimens

### Enrichment
- `reference_sample_processing_protocol` could be fluorescence-activated cell sorting for DAPI- cells. You can use it connect together all the cells pooled for `reference_sample` and add "During FACS isolation, DAPI-positive cells were excluded to remove dead and damaged cells. " to the description

### Cell suspension
- I would add a cell suspension for the cell lines and use `dissociation_protocol_cell_line` for the dissociation protocol
- `reference_sample` can have all the colon, PBMCs and cell line suspensions as input and `reference_sample_processing_protocol` for enrichment protocol 

### Library prep
- `library_preparation_protocol_C1` according to the supplemetary material this technique is barcoded. The GEO accession suggests  R1 = BC (1-6), UMI (7-11) but I have not found anything to confirm that. The end bias should be 3' and the strand might be first.
- `library_preparation_protocol_ICELL8` these are the barcodes length according to geo R1 = BC (1-11), UMI (12-22); R2 = cDNA
- `library_preparation_protocol_MARSseq` these are the barcodes length according to geo R1 = cDNA; R2 = BC (1-6), UMI (7-14)
- `library_preparation_protocol_QuartzSeq2` these are the barcodes length according to geo R1 = BC (1-15), UMI (16-23), R2 = cDNA
- `library_preparation_protocol_gmSCRBseq` these are the barcodes length according to geo R1 = BC (1-6), UMI (7-16); R2 = cDNA 
- `library_preparation_protocol_ddSEQ` these are the barcodes length according to geo R1 = BC (8-14,25-31,42-48), UMI (1-7); R2 = cDNA


### Sequencing protocol
- `sequencing_protocol_C1` should also be tag-based
- `sequencing_protocol_CELSeq2` should have HiSeq 3000 according to GEO
- `sequencing_protocol_ICELL8` should have HiSeq 4000 according to GEO
- `sequencing_protocol_gmSCRBseq` should have HiSeq 2000 according to GEO
- `sequencing_protocol_inDrop` should have HiSeq 4000 according to GEO

### Sequence files
- smartSeq2 is multiplexed so we need I1 and I2 to demultiplex, are they available?

### Analysis protocol
- the protocol description contains the numbered references from the publication
- the computational method field is a copy of derivation of process and it contains double pipes
- `analysis_protocol_processed`says the data was normalised but the counts in the file don't look like they have been normalised. Is this the protocol applied to the data available from geo? If not it can be deleted

### Analysis file
- the content description for metadata files should not be gene expression matrix
- you could add the matrix cell count
- the analysis matrix for the metadata files should be `analysis_protocol_raw`

**Comment by ami-day** [comment]
Test data is here: https://docs.google.com/spreadsheets/d/1CQ6e2AiJin5JaDAMfeu5poqDB-9y2wXf0s3AYJeFbiQ/edit#gid=513593862

**Comment by ami-day** [comment]
Need to convert [SRR9621775](https://sra-pub-run-odp.s3.amazonaws.com/sra/SRR9621775/SRR9621775) to fastq

**Comment by ami-day** [comment]
Waiting on this PR:
https://github.com/HumanCellAtlas/schema-test-data/pull/28

**Comment by Wkt8** [comment]
Unblocked by cell-to-cell-suspension test data.

**Comment by gabsie** [comment]
needs a look from another wrangler on the modelling  - maybe @anu-shiva and @Wkt8

**Comment by idazucchi** [comment]
two additional issues:
- [ ] smartSeq2 has both I1 and I2 and that is not allowed by the graph validation
- [ ] cell size selection for 10x only, apply to sequence file??

**Comment by idazucchi** [comment]
edit `smartseq2_has_2-3_files`
```
COUNT(f.`read_index`) as num_files, COUNT(DISTINCT f.`read_index`) as read_set
AND NOT num_files = read_set
RETURN p, "Smart-Seq2 protocols should contain between 2 and 3 files", labels(p)
```
? f is all files? I need just the ones linked to the process

**Comment by gabsie** [comment]
blocked by the graph validation issue

**Comment by gabsie** [comment]
Import form is filled
